robot-command/src/main/java/org/obolibrary/robot/FilterCommand.java

```diff
@@ -43,6 +43,8 @@ public FilterCommand() {
     o.addOption("r", "trim", true, "if true, keep axioms containing only selected objects");
     o.addOption(
         "S", "signature", true, "if true, keep axioms with any selected entity in their signature");
+    o.addOption(
+        "d", "drop_axiom_annotation", true, "annotation property to drop axiom annotations");
```

Comment:
kebap

Reply:
"annotation property to drop axiom annotations" -> "drop all axiom annotations involving a particular annotation property"

Reply:
Fixed